manager.go

```diff
@@ -376,12 +409,17 @@ func (m *Manager) dial(ctx context.Context) error {
 		dialOpts = append(dialOpts, rpc.WithInsecure())
 	}
 
-	conn, err := rpc.DialDirectGRPC(ctx, u.Host, m.logger, dialOpts...)
+	conn, err := rpc.DialDirectGRPC(ctx, u.Host, m.logger.AsZap(), dialOpts...)
 	if err != nil {
 		return err
 	}
 	m.conn = conn
 	m.client = pb.NewAgentDeviceServiceClient(m.conn)
+
+	// todo: ideally we would create the NetAppender ASAP (so logs are captured) and only *connect* it here.
```

Comment:
Is this something you're working on for this PR? If not, we (and I include myself here) should get in the habit of making all long term TODOs into jira tickets and linking them. E.g. `TODO(RSDK-9999): blah blah blah`

Reply:
done 5312398